doc/source/timedeltas.rst

```diff
@@ -363,6 +363,12 @@ or ``np.timedelta64`` objects. Passing ``np.nan/pd.NaT/nat`` will represent miss
    pd.TimedeltaIndex(['1 days', '1 days, 00:00:05',
                      np.timedelta64(2,'D'), datetime.timedelta(days=2,seconds=2)])
 
+   In [5]: pd.TimedeltaIndex(['0 days', '10 days', '20 days'])
```

Comment:
I think you can remove this first example, since the existing example above is similar.  I'm open to leaving this in though, if you'd rather keep this as a second example.